Configuration/Eras/python/Era_Run2_2017_HCALdev_cff.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
+from Configuration.Eras.Modifier_run2_HE_2017_cff import run2_HE_2017
+from Configuration.Eras.Modifier_run2_HF_2017_cff import run2_HF_2017
+from Configuration.Eras.Modifier_run2_HCAL_2017_cff import run2_HCAL_2017
+
+Run2_2017_HCALdev = cms.ModifierChain(Run2_2016, phase1Pixel, trackingPhase1, run2_HE_2017, run2_HF_2017, run2_HCAL_2017)
```

Comment:
@davidlange6 Could you clarify what you mean with "divergence"? The above is equivalent to
```
Run2_2017_HCALdev = cms.ModifierChain(Run2_2017, run2_HF_2017, run2_HCAL_2017)
```
which to me looks consistent with the workflow definition in this PR (or maybe your comment 2 was related to comment 1?).

Otherwise I agree with your comment 1 ("we agreed to have the new pixel config with the hcaldev scenario"), i.e. the new workflow should be "pre11 HCAL" and the new HCAL should be put in the default "2017" workflow.